lib/galaxy/web/framework/webapp.py

```diff
@@ -928,6 +929,27 @@ def fill_template_string(self, template_string, context=None, **kwargs):
         return str(template)
 
 
+def build_native_uwsgi_app( paste_factory, config_section ):
```

Comment:
https://github.com/galaxyproject/galaxy/blob/69ff6ab906d94fdb4da3bd0b858fce2de3b349fa/lib/galaxy/webapps/galaxy/buildapp.py#L149 should probably be updated to point at this function.